docs/userguide/datafiles.rst

```diff
@@ -96,31 +233,189 @@ the ``setup.py`` ``package_data`` list is updated before calling ``setup.py``.
 
 .. https://docs.python.org/3/distutils/setupscript.html#installing-package-data
 
+exclude_package_data
+====================
+
 Sometimes, the ``include_package_data`` or ``package_data`` options alone
-aren't sufficient to precisely define what files you want included.  For
-example, you may want to include package README files in your revision control
-system and source distributions, but exclude them from being installed.  So,
-setuptools offers an ``exclude_package_data`` option as well, that allows you
-to do things like this::
+aren't sufficient to precisely define what files you want included. For example,
+consider a scenario where you have ``include_package_data=True``, and you are using
+a revision control system with an appropriate plugin. Your README is probably being
```

Comment:
Maybe we should think a bit better about this example (inherited from the previous version of this guide).

Most of the times `README` files are placed outside of the package directory, so the example might confuse new Python users...